roles/build-release/tasks/tests.yaml

```diff
@@ -24,101 +24,101 @@
-        {{ antsibull_ansible_venv }}/bin/python3 -c 'from ansible_collections.ansible_release import ansible_version; print(ansible_version)'
-      changed_when: false
-      register: _ansible_version_builtin
-
-    - name: Validate the version of ansible
-      ansible.builtin.assert:
-        that:
-          - antsibull_venv_pip_ansible_version == _deps['_ansible_version']
-          - antsibull_venv_pip_ansible_version == _ansible_version_builtin.stdout
-        success_msg: "ansible {{ antsibull_venv_pip_ansible_version }} matches {{ _deps_file }} as well as 'ansible_collections.ansible_release'"
-        fail_msg: "ansible {{ antsibull_venv_pip_ansible_version }} does not match {{ _deps_file }} or 'ansible_collections.ansible_release'"
-
-    - when: antsibull_ansible_version is _antsibull_packaging_version('6.0.0rc1', '>=')
-      block:
-        - name: Retrieve the builtin reported version of ansible from the ansible-community CLI tool
+- vars:
+    antsibull_venv_pip_bin: "{{ antsibull_ansible_venv }}/bin/pip"
+    antsibull_venv_pip_pkgs: "{{ _pip_pkgs['packages'][antsibull_venv_pip_bin] }}"
+    antsibull_venv_pip_ansible_version: "{{ antsibull_venv_pip_pkgs['ansible'][0]['version'] }}"
+    antsibull_venv_pip_ansible_core_version: "{{ antsibull_venv_pip_pkgs['ansible-core'][0]['version'] }}"
+  block:
+    # Note: the version of ansible-core doesn't necessarily match the deps file since the version requirement is >=
+    - block:
+        - name: Load deps_file
+          ansible.builtin.set_fact:
+            _deps: >-
+              {{ lookup('file', antsibull_data_dir ~ '/' ~ _deps_file) | _antsibull_parse_deps }}
+
+        - name: Query installed pip packages
+          community.general.pip_package_info:
+            clients: "{{ antsibull_venv_pip_bin }}"
+          register: _pip_pkgs
+
+        - name: Validate the version of ansible-core
+          ansible.builtin.assert:
+            that:
+              - antsibull_venv_pip_ansible_core_version is _antsibull_packaging_version(antsibull_expected_ansible_core, '>=')
+            success_msg: "ansible-core {{ antsibull_venv_pip_ansible_core_version }} matches (or exceeds) {{ _deps_file }}"
+            fail_msg: "ansible-core {{ antsibull_venv_pip_ansible_core_version }} does not match {{ _deps_file }}"
+          vars:
+            antsibull_expected_ansible_core: "{{ _deps['_ansible_core_version'] }}"
+
+    - block:
+        - name: Retrieve collections that should be included in the package
+          shell: cat {{ antsibull_data_dir }}/ansible.in | grep -vE "^#"
+          changed_when: false
+          register: _included_collections
+
+        - name: Retrieve the builtin reported version of ansible
           ansible.builtin.command: >-
-            {{ antsibull_ansible_venv }}/bin/ansible-community --version
+            {{ antsibull_ansible_venv }}/bin/python3 -c 'from ansible_collections.ansible_release import ansible_version; print(ansible_version)'
```

Comment:
What's the purpose of this change? Do we not want to test the CLI entrypoint?

Reply:
This line didn't change, it's mainly a problem how the diff is represented. I changed indentation of a larger block of tasks, and there are two tasks for retrieving the version (one registers `_ansible_version_builtin`, the other `_ansible_version_builtin_2`). The GitHub diff viewer now shows this as if one of them is transformed to the other.